packages/experimental/Switch/src/Switch.types.ts

```diff
@@ -110,6 +110,7 @@ export interface SwitchTokens extends LayoutTokens, FontTokens, IBorderTokens, I
 
   /**
    * States that can be applied to a switch
+   * Note: 'hovered','focused','before','beforeContent','above' are not supported for Android
```

Comment:
Please update similar comments in SPEC.md

Reply:
Updated.